client/src/plus/app.tsx

```diff
@@ -146,196 +149,219 @@ export default function App() {
+                    solid background color for really old browsers, and a
+                    semi-transparent color for newer browsers:
+                  </p>
+                  <div className="code-snippet">
+                    <code>
+                      <span className="code-c">background-color</span>: red;
+                      <br />
+                      <span className="code-c">background-color</span>: rgba(
+                      <span className="code-m">255</span>,
+                      <span className="code-m">0</span>,
+                      <span className="code-m">0</span>,
+                      <span className="code-m">0.6</span>);
+                    </code>
                   </div>
                 </div>
-                <h4>Feature Queries</h4>
-                <p>
-                  For very simple fallbacks, the overriding method shown
-                  previously may work. It can however require that you order the
-                  declarations and rules in your CSS carefully, making it more
-                  brittle than you might like. You may also run into problems
-                  when you want to use additional CSS to enhance the layout in
-                  newer browsers, if that CSS is also understood by older
-                  browsers.
-                </p>
-                <p>
-                  In the next demo, I have given the left-hand column a
-                  background color. I only want this to apply to the CSS Grid
-                  layout, where I can ensure that the columns will be the same
-                  height as each other. However, using the previous method the
-                  background color is understood and therefore used by browsers
-                  without CSS Grid support too. I have also added widths to the
-                  floated elements. As a percentage width is interpreted by the
-                  grid layout as a percentage of the column track, this causes
-                  the columns to become narrower than the track.
-                </p>
-                <div className="code-snippet">
-                  <div className="codepen">
-                    <iframe
-                      id="cp_embed_MWJVaqm"
-                      src="https://codepen.io/rachelandrew/embed/gOmrXqQ?height=450&amp;theme-id=1&amp;slug-hash=gOmrXqQ&amp;default-tab=css,result"
-                      scrolling="no"
-                      allowFullScreen={false}
-                      title="Modern CSS 2.3 before"
-                      className="codepen"
-                      loading="lazy"
-                      style={{
-                        width: "100%",
-                        overflow: "hidden",
-                        height: "100%",
-                      }}
-                      frameBorder={0}
-                    ></iframe>
+                <div
+                  style={
+                    showDeepDive ? { display: "block" } : { display: "none" }
```

Comment:
Why this? Why render it at all? You could just do:
```jsx
{showDeepDive && <div> ...... </div>}
```
instead of using React to CSS-hide things.

Reply:
I'll change this, I wasn't aware I could do this. Thanks!